lti_consumer/lti_consumer.py

```diff
@@ -740,6 +799,85 @@ def student_view(self, context):
         fragment.initialize_js('LtiConsumerXBlock')
         return fragment
 
+    has_author_view = True
+
+    def author_view(self, context):
+        """
+        XBlock studio view of this component.
+
+        Makes a request to `lti_launch_handler` either
+        in an iframe or in a new window depending on the
+        configuration of the instance of this XBlock
+
+        Arguments:
+            context (dict): XBlock context
+
+        Returns:
+            xblock.fragment.Fragment: XBlock HTML fragment
+        """
+        if self.lti_version != "lti_1p3":
+            return self.student_view(context)
```

Comment:
Just trying to recall from our conversations - why is it we need a new author_view but didn't need one for v1.1?  Is it because we're displaying a preview of the tool in studio, which we didn't do before?

Reply:
So the `author_view` here is being used to display a few variables that the user need to set up in the LTI Tool in order for the launch to work.
Furthermore, launches can't be performed from the Studio since the tool will need to access LMS URLs.

This is the Studio view with this change when using LTI 1.3: 
![image](https://user-images.githubusercontent.com/27893385/80490657-12837380-8938-11ea-9651-1e5c8816992e.png)

The LTI 1.1 works as usual and displays the launch preview.